scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/struct/LazilyGeneratedMethodParameterData.java

```diff
@@ -0,0 +1,160 @@
+
+package org.scijava.ops.engine.struct;
+
+import com.github.therapi.runtimejavadoc.MethodJavadoc;
+import com.github.therapi.runtimejavadoc.OtherJavadoc;
+import com.github.therapi.runtimejavadoc.RuntimeJavadoc;
```

Comment:
scijava/scijava#89

Reply:
Resolving this in favor of discussion in that issue.